<launch>
	<arg name="marker_size" default="7" />
	<arg name="max_new_marker_error" default="0.08" />
	<arg name="max_track_error" default="0.2" />

	<arg name="cam_image_topic" default="/head_mount_kinect/depth_registered/points" />
	<arg name="cam_info_topic" default="/head_mount_kinect/rgb/camera_info" />		
	<arg name="output_frame" default="/torso_lift_link" />

         <include file="$(find openni_launch)/launch/openni.launch">
          <arg name="depth_registration" value="true"/>
          <arg name="camera" value="head_mount_kinect"/>
          <arg name="rgb_frame_id" value="/head_mount_kinect_rgb_optical_frame"/>
          <arg name="depth_frame_id" value="/head_mount_kinect_depth_optical_frame"/>
          <arg name="publish_tf" value="false"/>
         </include >

	<node name="ar_track_alvar" pkg="ar_track_alvar" type="individualMarkers" respawn="false" args="$(arg marker_size) $(arg max_new_marker_error) $(arg max_track_error) $(arg cam_image_topic) $(arg cam_info_topic) $(arg output_frame)" />
</launch>
